pb/sigchain.go

```diff
@@ -155,11 +156,9 @@ func NewSigChain(srcPubKey *crypto.PubKey, srcPrivKey []byte, dataSize uint32, d
 	}
 
 	hash := sha256.Sum256(buff.Bytes())
-	signature, err := crypto.Sign(srcPrivKey, hash[:])
-	if err != nil {
-		return nil, err
-	}
+	signature, proof := crypto.GenerateVrf(srcPrivKey, hash[:])
```

Comment:
since there is no error returned, what if srcPrivKey is invalid?

Reply:
Only need to be 32 bytes.

Reply:
done